src/Microsoft.Build.Sql/Microsoft.Build.Sql.csproj

```diff
@@ -85,6 +85,7 @@
 
   <ItemGroup>
     <Content Include="sdk\**" Pack="true" PackagePath="sdk\" />
+    <Content Include="tools\**" Pack="true" PackagePath="tools\" />
```

Comment:
This is included on line 61 after restoring package references from DacFx, but that target may not run if build and pack are separate targets, causing the Nuget package to not contain any of the DacFx DLLs.